components/style/gecko_bindings/bindings.rs

```diff
@@ -518,6 +518,12 @@ extern "C" {
     pub fn Gecko_DestroyClipPath(clip: *mut StyleClipPath);
 }
 extern "C" {
+    pub fn Gecko_ResetFilters(effects: *mut nsStyleEffects, len: usize);
+}
+extern "C" {
+    pub fn Gecko_CopyFiltersFrom(aSrc: &nsStyleEffects, aDest: *mut nsStyleEffects);
```

Comment:
I'm pointing the `/path/to/objdir` to my `stylo-incubator` one, should I use the latest mozilla-central instead?